autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
@jamessan I'm referring to my experience with it, which I imagine isn't unique.  Personally, I never read the doc on `:verbose` until a few days ago.  It was something I picked up along the way.  Everything I regularly use it with (`:verbose nmap`, `:verbose autocmd`, etc) appears to give you the most verbose output.  `:100verbose autocmd` doesn't make it any more informative, so I would assume that's the case for everything.

Only things I saw in the docs where the levels mattered is for debugging VimL execution, and it looks like the levels are for filtering message origins (autocmd execution, diff, exceptions), not message types.

From my point of view, `:verbose Neomake` would be something a developer asks a user to do for troubleshooting.  Wouldn't we want it to be as verbose as possible in that case?  `g:neomake_verbose` then would be for testing and passive logging in development.